docs/OLED.md

```diff
@@ -0,0 +1,139 @@
+# mLRS Documentation: OLED Display #
+
+([back to main page](../README.md))
+
+mLRS allows for a 0.96", 128x64, SSD1306 OLED display to be connected to a Tx module using I2C.  When paired with a 5-way joystick (4 directions + select) this allows for one to update parameters and display link statistics.  The OLED is particularly useful on radios that do not have a 480 x 272 color LCD and cannot utilize the color Lua script.
+
+## Hardware Configuration
+
+Instructions for adding an OLED display to the FRM303 module [here](https://github.com/olliw42/mLRS-docu/blob/master/docs/FLYSKY_FRM303.md#oled-display-addition).
+
+## OLED Screens & Navigation
+
+From the main screen, you can navigate the through 4 main screens using the up and down directions, these contain the following information:
```

Comment:
shouldn't it be "... navigate through ... " 
maybe also a '.' before "these conatin..."

Reply:
Good catch, updated.